### DISCUSSION TITLE
有群监听不到

### DISCUSSION THREAD

**Discussion by yujumpjump** [simple-robot/Discussions#90] [Q&A]
### 涉及的编程语言

Java

### 疑问描述

在没有使用任何过滤条件的注解 监听不到群里面的事件（之前是可以监听到的） ，但是其他的群可以监听到

**Comment by yujumpjump** [comment]
已经解决了，重新退群 在加就解决了

**Comment by yujumpjump** [comment]
还是这个问题 过一会群还是监听不到了 但是另一个群可以监听到 我不知道什么情况

**Reply by ForteScarlet** [reply]
首先确定是 **无法触发监听函数** 还是 **无法发送消息/消息发送无效** 。如果是后者，大概率风控，如果是前者，反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)
